client/src/App.js

```diff
@@ -87,6 +92,14 @@ class App extends Component {
             render={() => <Shop loggedIn={this.state.loggedIn} />}
           />
         </div>
+        <Footer
+          copyrights="&copy; 2019 Copyright Piggy Business"
+          moreLinks={
```

Comment:
you're sending a whole anchor tag?? why not just send down the href text link and then in the footer have a <a></a> tag in the component because the element is a part of the component and sending down the whole element down like this is hard to find and edit in the footer when actually it's in the App that gets sent down to the footer.